packages/client/test/client.test.js

```diff
@@ -589,6 +590,27 @@ describe('PercyClient', () => {
         }
       });
     });
+
+    it('can upload a resource from a local path in GZIP format', async () => {
+      process.env.PERCY_GZIP = true;
```

Comment:
this should be moved to before and after block so that it can be cleared after current test.